Sloth.Core/Services/CyberSourceBankReconcileService.cs

```diff
@@ -169,29 +169,57 @@ await using (var tran = await _context.Database.BeginTransactionAsync())
                             CybersourceBankReconcileJobRecordId = jobRecord.Id
                         }.SetStatus(TransactionStatuses.Scheduled);
 
-                        // move money out of clearing
-                        var clearing = new Transfer()
+                        if (!AccountValidationService.IsKfsAccount(integration.ClearingAccount))
```

Comment:
would be nice to invert the IF, so the negative case isn't first, but otherwise this looks good.